NebulaWorld/MonoBehaviours/Remote/RemotePlayerAnimation.cs

```diff
@@ -36,7 +36,7 @@ public void UpdateState(PlayerAnimationUpdate packet)
             PlayerAnimator.jumpWeight = packet.JumpWeight;
             PlayerAnimator.jumpNormalizedTime = packet.JumpNormalizedTime;
             PlayerAnimator.idleAnimIndex = packet.IdleAnimIndex;
-            PlayerAnimator.sailAnimIndex = packet.SailAnimIndex;
+            PlayerAnimator.sailAnimIndex = 0;
```

Comment:
Won't this stop animating flying remote clients?

Reply:
SailAnimIndex is used when mecha is flying intersteallar. 0 = normal, 3 = wobbling effect of wrapping, 1 & 2 is related to a CosmicMessage.
I'm not sure if there is a way for a player to see another player closely during sailing in the current game, so I disable it. Will add the syncing back if it is possible.

Reply:
Thanks for the clarification!